docs/deployment-arbitrary-cluster.md

```diff
@@ -0,0 +1,124 @@
+# Deployment on Arbitrary Cluster
+
+In this guide we will cover installation of Open vSwitch, NetworkManager and
+kubernetes-nmstate on your arbitrary cluster.
+
+## Requirements
+
+This guide requires you to have your own Kubernetes/OpenShift cluster. If you
+don't have one and just want to try kubernetes-nmstate out, please refer to
+[deployment on local cluster](deployment-local-cluster.md) guide.
+
+In order to get kubernetes-nmstate running, NetworkManager, and optionally
+Open vSwitch, must be installed on node. This requirement will be dropped once
+we make these components a part of kubernetes-nmstate container images.
+
+### NetworkManager
+
+kubernetes-nmstate container communicates with a NetworkManager instance running
+on the node using D-Bus. Make sure that NetworkManager is installed and running
+on each node.
+
+On CentOS:
+
+```shell
+yum install openvswitch
+systemctl start openvswitch
+```
+
+On Fedora:
+
+```shell
+dnf install openvswitch
+systemctl start openvswitch
+```
+
+### Open vSwitch
+
+This part is optional. If you want to control Open vSwitch interfaces using
+nmstate, Open vSwitch and its NetworkManager plugin must be installed on the
+node.
+
+#### Kubernetes
+
+With Kubernetes, all you have to do is to install openvswitch package and its
+NetworkManager plugin.
+
+On CentOS (with a new Open vSwitch version, not shipped in default repositories):
```

Comment:
Instead of `with...` it should state that one needs at least a version 2.9.2 and it should also explain what this package source is.

Reply:
Done